GomokuClient/packages/gomoku-core/src/pages/HomeGame.tsx

```diff
@@ -20,6 +21,16 @@ export const HomeGame = () => {
   const { jwtToken } = useAuthToken();
   const { data: paginatedGames } = useFetchGames(jwtToken);
   const { data: paginatedActiveGames } = useFetchActiveGames(jwtToken);
+  const { hubProxy, isConnected } = useSignalRConnection();
+
+  useEffect(() => {
+    if (isConnected && hubProxy) {
+      hubProxy.sendInvitationToPlay({
+        //TODO: Remove. This is for testing
+        playerId: "test",
+      });
+    }
+  }, [isConnected, hubProxy]);
```

Comment:
Can you give explanation why we need this?

Reply:
Leftover. Tested, that I can send event in home screen. Will remove